django_mailbox/models.py

```diff
@@ -18,13 +18,13 @@
 import uuid
 
 import six
-from six.moves.urllib.parse import parse_qs, unquote, urlparse
```

Comment:
These `six` imports are 100% correct; see here: https://pythonhosted.org/six/#module-six.moves .  What sort of problem are you running into when using them?